esmvaltool/cmorizers/obs/utilities.py

```diff
@@ -120,6 +120,7 @@ def flip_dim_coord(cube, coord_name):
     coord_idx = cube.coord_dims(coord)[0]
     coord.points = np.flip(coord.points)
     coord.bounds = np.flip(coord.bounds, axis=0)
+    coord.bounds = np.flip(coord.bounds, axis=1)
```

Comment:
This is still here, did you want to change this back, as you discussed with @schlunma?

Reply:
Yes, indeed. Thanks for spotting this.